board-games/test/game-night-test.js

```diff
@@ -0,0 +1,129 @@
+var Game = require('../src/board-game.js');
+var GameNight = require('../src/game-night.js');
+
+describe ('GameNight', function() {
+  it.skip('should have a host', function() {
+    var host = new Player({name: 'Elizabeth'});
+    var gameNight = new GameNight(host);
+
+    assert.isFunction(GameNight);
+    assert.equal(gameNight.host.name, 'Elizabeth');
+  });
+
+  it.skip('should be able to invite other gamers', function() {
+    var host = new Player({name: 'Elizabeth'});
+    var player1 = new Player({name: 'Phil'});
+    var player2 = new Player({name: 'Ryan'});
+    var gameNight = new GameNight(host);
+
+    gameNight.invite([player1, player2]);
+
+    assert.equal(gameNight.players.length, 3);
+    assert.equal(gameNight.players[0].name, 'Elizabeth');
+    assert.equal(gameNight.players[2].name, 'Ryan');
+  });
+
+  // All games in the hosts' collection should be available to play
+  it.skip('should be able to assemble a collection of games to choose from', function() {
+    var host = new Player({name: 'Elizabeth'});
+    var player1 = new Player({name: 'Katie'});
+    var player2 = new Player({name: 'Hannah'});
+    var game1 = new Game({name: 'Mysterium'});
+    var game2 = new Game({name: 'Callico'});
+    var game3 = new Game({name: 'Wingspan'});
+    var game4 = new Game({name: 'Pandemic'});
+    var game5 = new Game({name: 'Gloomhaven: Jaws of the Lion'});
+    var game6 = new Game({name: 'Sagrada'});
+    var game7 = new Game({name: 'Cards Against Humanity'});
+    var game8 = new Game({name: 'Carcassonne'});
+
+    host.addToCollection(game1);
+    host.addToCollection(game2);
+    host.addToCollection(game3);
+    host.addToCollection(game4);
+    player1.addToCollection(game5);
+    player2.addToCollection(game6);
+    player2.addToCollection(game7);
+    player2.addToCollection(game8);
+    player1.takeToGameNight([game5]);
+    player2.takeToGameNight([game6, game8]);
+
+    var gameNight = new GameNight(host);
+
+    gameNight.invite([player1, player2]);
+    gameNight.makeCollection();
+
+    assert.equal(gameNight.games.length, 7);
+    assert.equal(gameNight.games[3].name, 'Pandemic');
+    assert.equal(gameNight.games[5].name, 'Sagrada');
+    assert.equal(gameNight.games[6].name, 'Carcassonne');
+  });
+  
+  it.skip('should be able to play a game and determine a winner', function() {
+    var host = new Player({name: 'Phil'});
+    var player1 = new Player({name: 'Paul'});
+    var player2 = new Player({name: 'Mack'});
+    var game = new Game({name: 'Nemesis'});
+    host.addToCollection(game);
+    
+    var gameNight = new GameNight(host);
+    gameNight.invite([player1, player2]);
+    gameNight.makeCollection();
+    gameNight.playAGame(game, 'Mack');
+    
+    assert.equal(host.experience, 1);
+    assert.equal(player1.experience, 1);
+    assert.equal(player1.wins[game.name], undefined);
+    assert.equal(player2.wins[game.name], 1);
+  });
+
+  it.skip('should be able to accommodate everyone', function() {
```

Comment:
I need a bit more info here, does this game not accomodate everyone because of the player count?

Reply:
I made a seperate chooseGame() method and changed the last few tests to accomodate the new method. I think this is more of what I was going for in the first place.